src/Controller/SlackController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Controller;
+
+use App\Notifiers\Slack\SlackNotifier;
+use App\Repository\ConferenceRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class SlackController extends AbstractController
+{
+    private SlackNotifier $slackNotifier;
+    private EntityManagerInterface $em;
+    private ConferenceRepository $conferenceRepository;
+    private HttpClientInterface $httpClient;
+
+    public function __construct(SlackNotifier $slackNotifier, EntityManagerInterface $em, ConferenceRepository $conferenceRepository, HttpClientInterface $httpClient)
+    {
+        $this->slackNotifier = $slackNotifier;
+        $this->em = $em;
+        $this->conferenceRepository = $conferenceRepository;
+        $this->httpClient = $httpClient ?? HttpClient::create();
+    }
+
+    /** @Route("/slack", name="slack_endpoint") */
+    public function slackEndPoint(Request $request): Response
+    {
+        $payload = json_decode($request->request->get('payload'));
+
+        $actionId = $payload->actions[0]->action_id;
+        $conference = $this->conferenceRepository->find($actionId);
+        $conference->setExcluded(true);
+
+        $this->em->persist($conference);
+        $this->em->flush();
+
+        $dailyConferences = $this->conferenceRepository->getDailyConferences();
+
+        $blocks = $this->slackNotifier->buildDailyBlocks($dailyConferences);
+
+        $body = [
+            'replace_original' => true,
+            'blocks' => $blocks,
+        ];
+
+        $this->httpClient->request('POST', $payload->response_url, [
+            'headers' => ['Content-type' => 'application/json'],
+            'body' => json_encode($body),
```

Comment:
```suggestion
            'json' => $body,
```

Reply:
You must defined a timeout. (in the application configuration is the best place I guess)